sdk/storage/azure_storage_blob/src/clients/blob_client.rs

```diff
@@ -230,7 +230,7 @@ impl BlobClient {
     /// * `options` - Optional configuration for the request.
     pub async fn set_tier(
         &self,
-        tier: AccessTierOptional,
+        tier: AccessTier,
```

Comment:
Does this type change count as a breaking change that should be called out?

Reply:
Hi @LarryOsterman good callout here. We never actually released with `AccessTierOptional`, we only got changed to that as a result of some efforts to get CI-clean in the feature/blob-tsp branch. 

We were trying to accommodate unblocking both CIs while iterating, but now that we are releasing another version of the SDK and no final decision has been made for `feature/blob-tsp`, we will revert to what we originally had (and released with in `v0.2.0`), knowing that it reintroduces a break in the `feature/blob-tsp` branch. This is understood and accepted as `feature/blob-tsp` has several other CI-blocking issues from getting that merged to `main`.

Please let me know if that you have any other questions!